client/build.gradle

```diff
@@ -1,105 +1,108 @@
+                "--ignore-file-override", "${projectDir}/.swagger-codegen-ignore",
+                "-D", "apiTests=false," +
+                      "apiDocs=false," +
+                      "modelTests=false," +
+                      "modelDocs=false," +
+                      "dateLibrary=java8"
+            ]
+        }
+    }
 }
 
 // Publishing
 
-// TODO: there may be a better way to supply these values than envvars.
 // This and the test below makes sure the build will fail reasonably if you try
 // to publish without the environment variables defined.
 def artifactory_username = System.getenv("ARTIFACTORY_USERNAME")
 def artifactory_password = System.getenv("ARTIFACTORY_PASSWORD")
 
 gradle.taskGraph.whenReady { taskGraph ->
-	if (taskGraph.hasTask(artifactoryPublish) &&
-			(artifactory_username == null || artifactory_password == null)) {
-		throw new GradleException("Set env vars ARTIFACTORY_USERNAME and ARTIFACTORY_PASSWORD to publish")
-	}
+    if (taskGraph.hasTask(artifactoryPublish) &&
+            (artifactory_username == null || artifactory_password == null)) {
+        throw new GradleException("Set env vars ARTIFACTORY_USERNAME and ARTIFACTORY_PASSWORD to publish")
+    }
 }
 
 publishing {
-	publications {
-		workspaceManagerClientLibrary(MavenPublication) {
-			from components.java
-			versionMapping {
-				usage("java-runtime") {
-					fromResolutionResult()
-				}
-			}
-		}
-	}
+    publications {
+        workspaceManagerClientLibrary(MavenPublication) {
+            from components.java
+            versionMapping {
+                usage("java-runtime") {
+                    fromResolutionResult()
+                }
+            }
+        }
+    }
 }
 
 artifactory {
-	publish {
-		contextUrl = "https://broadinstitute.jfrog.io/broadinstitute/"
-		repository {
-			repoKey = "libs-snapshot-local" // The Artifactory repository key to publish to
-			username = "${artifactory_username}" // The publisher user name
-			password = "${artifactory_password}" // The publisher password
-		}
-		defaults {
-			// This is how we tell the Artifactory Plugin which artifacts should be published to Artifactory.
-			// Reference to Gradle publications defined in the build script.
-			publications("workspaceManagerClientLibrary")
-			publishArtifacts = true
-			publishPom = true
-		}
-	}
+    publish {
+        contextUrl = "https://broadinstitute.jfrog.io/broadinstitute/"
+        repository {
+            repoKey = "libs-snapshot-local" // The Artifactory repository key to publish to
+            username = "${artifactory_username}" // The publisher user name
+            password = "${artifactory_password}" // The publisher password
+        }
+        defaults {
+            // This is how we tell the Artifactory Plugin which artifacts should be published to Artifactory.
+            // Reference to Gradle publications defined in the build script.
+            publications("workspaceManagerClientLibrary")
```

Comment:
Is there a way to publish source from here?

Reply:
Yes, I'll add that.